modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -126,10 +138,78 @@ gadmin config apply
 
 After the parameters are successfully configured, you can access your UDFs in new queries right away.
 
+=== Upload a local UDF file through `gadmin`
+
+You can configure `gadmin` to upload a local file to the GSQL server.
+Once uploaded, the UDF file is available to all nodes in a TigerGraph cluster.
+Moving the local file after uploading does not affect the UDF available to GSQL.
+
+==== Modify current query UDF file
+
+Run the following command to retrieve the current UDF file to a specified location on the server.
+Replace `<path_to_file>` with the filepath where you want the UDF file:
+
+[.wrap,console]
+----
+$ gadmin config get GSQL.UDF.ExprFunctions  >  <path_to_file> <1>
+$ gadmin config get GSQL.UDF.ExprUtil > <path_to_file> <2>
+----
+<1> This retrieves `ExprFunctions` where the UDFs are defined.
+<2> This retrieves `ExprUtil`, where helper functions are defined.
+
+For example:
+
+[.wrap,console]
+----
+$ gadmin config get GSQL.UDF.ExprFunctions  >  /home/tmpExprFunctions.hpp
+----
+
+==== Define your function
+
+Write your function in ExprFunctions and any helper functions in ExprUtil.
+
+[CAUTION]
+====
+If any code in `ExprFunctions.hpp` or `ExprUtil.hpp` causes a compilation error, GSQL will be unable to install _any_ new queries, whether containing user-defined functions or not.
+====
+
+==== Upload query UDF file
+
+After modifying the file, run the following command to upload the modified file.
+Replace `<path_to_file>` with the path of the UDF file:
 
-== Store a UDF file locally
+[.wrap,console]
+----
+$ gadmin config set GSQL.UDF.ExprFunctions   @<path_to_file>
+----
+
+
+For example:
+
+[.wrap,console]
+----
+$ gadmin config set GSQL.UDF.ExprFunctions   @/home/tmpExprFunctions.hpp
+----
```

Comment:
I suggest to also append `gadmin config apply` command in the example similar to the Github example above.
Also, similar to gadmin setting in Github example, specify that users need to restart GSQL if it is still running to make changes work.